bin/p2-rctl/main.go

```diff
@@ -104,7 +105,6 @@ func main() {
 	}
 
 	httpClient := cleanhttp.DefaultClient()
-	opts.Client = httpClient
```

Comment:
I'll happily take alternatives that allow me to use this in `opts.Client` (just need to be able to set headers based on our command line flags), but I don't see any in  https://github.com/hashicorp/go-cleanhttp/blob/master/cleanhttp.go